tailwind.config.ts

```diff
@@ -202,12 +202,26 @@ const config: Config = {
       colors: colorPalette,
       backgroundImage: {
         'bottom-border-glow': `radial-gradient(circle at center, ${colorPalette.anakiwa} 0, blue, transparent 100%)`,
-        'button-gradient':
-          'linear-gradient(283deg, #0808A5 0%, rgba(8, 8, 165, 0.00) 100%)',
-        'button-gradient-hover':
-          'linear-gradient(283deg, #44F 0%, rgba(23, 23, 255, 0.10) 100%)',
-        'button-gradient-focus':
-          'linear-gradient(283deg, #0D0DD2 0%, rgba(23, 23, 255, 0.10) 100%)'
+        // Primary
+        'button-gradient': 'linear-gradient(283deg, #0808A5 0%, #0808a500 100%)',
```

Comment:
This feels messy to me - im not sure there is a better way but wanted to call it out.

Reply:
I don't have idea how to make it better

Reply:
you could experiment with somethin like `filled: bg-gradient-to-r from-transparent to-primary hover:brightness-125` instead of having all these custom classes. It looks pretty much the same